packages/components/src/AspectRatio.js

```diff
@@ -18,7 +23,7 @@ export const AspectRatio = React.forwardRef(
       />
       <Box
         {...props}
-        __css={{
+        sx={{
```

Comment:
I'd be considering changing the private API a bit here, but does this mean that `props.sx` is no longer forwarded to the component here? This looks like it'd break in MDX usage

Reply:
Yes you're right, sorry had overlooked that, I've re-added the `__css` API.